lib/jsdom/level2/html.js

```diff
@@ -1270,6 +1270,14 @@ define('HTMLLegendElement', {
   proto: {
     get form() {
       return closest(this, 'FORM');
+    },
+    click: function() {
```

Comment:
No, there is only one `click()` method in the HTML spec. Basically HTML uses spec-level polymorphism instead of JavaScript-level polymorphism :-/

Good point about button though; that means my above test is not quite right.